apps/team-awesome-web-e2e/src/e2e/app.cy.ts

```diff
@@ -1,13 +1,26 @@
-import { getGreeting } from '../support/app.po';
+describe("Routing", () => {
+  beforeEach(() => cy.visit("/"));
 
-describe('team-awesome-web', () => {
-  beforeEach(() => cy.visit('/'));
+  it("Navigates to other pages", () => {
+    cy.viewport(1400, 800);
+    cy.visit("http://localhost:4200");
 
-  it('should display welcome message', () => {
-    // Custom command example, see `../support/commands.ts` file
-    cy.login('my-email@something.com', 'myPassword');
+    // team-members page
+    cy.get("[data-cy=nav-team-members]").click();
 
-    // Function helper example, see `../support/app.po.ts` file
-    getGreeting().contains('Welcome team-awesome-web');
+    cy.get("[data-cy=nav-timesheets]").click();
+  });
+
+  it("Navigates to other pages on mobile", () => {
+    cy.viewport(550, 750);
+    cy.get("[data-cy=nav-button]").click();
+
+    cy.get("[data-cy=nav-team-members]").click();
+
+    cy.wait(1000);
```

Comment:
While this might be effective now, it is dependent on the page load. Can you try for about 10 minutes to find a different way to solve this so that it's more directly linked to the actual loading

Reply:
Resolved with cy.url().should("include", "teammembers")